packages/battery/battery/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -229,22 +216,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "/bin/sh \"$FLUTTER_ROOT/packages/flutter_tools/bin/xcode_backend.sh\" thin";
-		};
-		4096151B6BA12D6D4D7DD96A /* [CP] Embed Pods Frameworks */ = {
```

Comment:
The Objective-C Podfiles are set up to use libraries, not frameworks.  CocoaPods is removing the build step that embeds the frameworks into the app.